parsing/parser.mly

```diff
@@ -745,6 +745,8 @@ let mk_directive ~loc name arg =
 %token DONE                   "done"
 %token DOT                    "."
 %token DOTDOT                 ".."
+%token DOTLESS                ".<"
```

Comment:
Should these also have the same comment as the `GREATERDOT`? In case anyone wondered why they are not used and decided to remove? `(* see the [metaocaml_expr] comment *)`

Reply:
Done, thanks!